subprojects/docs/src/docs/userguide/writing_build_scripts.adoc

```diff
@@ -96,153 +101,195 @@ The `Project` object provides some standard properties, which are available in y
+
+[source.multi-language-sample,groovy]
+.build.gradle
+----
+include::{samplesPath}/userguide/tutorial/extraProperties/build.gradle[tag=extraProperties]
+----
+
+.Output of **`gradle -q printProperties`**
+----
+> gradle -q printProperties
+include::{samplesPath}/userguide/tutorial/extraProperties/extraProperties.out[]
+----
 
 In this example, an `ext` block adds two extra properties to the `project` object. Additionally, a property named `purpose` is added to each source set by setting `ext.purpose` to `null` (`null` is a permissible value). Once the properties have been added, they can be read and set like predefined properties.
 
 By requiring special syntax for adding a property, Gradle can fail fast when an attempt is made to set a (predefined or extra) property but the property is misspelled or does not exist. Extra properties can be accessed from anywhere their owning object can be accessed, giving them a wider scope than local variables. Extra properties on a project are visible from its subprojects.
 
-For further details on extra properties and their API, see the api:org.gradle.api.plugins.ExtraPropertiesExtension[] class in the API documentation.
+For further details on extra properties and their API, see the link:{groovyDslPath}/org.gradle.api.plugins.ExtraPropertiesExtension.html[ExtraPropertiesExtension] class in the API documentation.
 
 [[sec:configuring_arbitrary_objects]]
-=== Configuring arbitrary objects
+== Configuring arbitrary objects
 
 You can configure arbitrary objects in the following very readable way.
 
-++++
-<sample xmlns:xi="http://www.w3.org/2001/XInclude" id="configureObject" dir="userguide/tutorial/configureObject" title="Configuring arbitrary objects">
-    <sourcefile file="build.gradle"/>
-    <output args="-q configure"/>
-</sample>
-++++
+=== Example: Configuring arbitrary objects
+
+[source.multi-language-sample,groovy]
+.build.gradle
+----
+include::{samplesPath}/userguide/tutorial/configureObject/build.gradle[]
+----
+
+.Output of **`gradle -q configure`**
+----
+> gradle -q configure
+include::{samplesPath}/userguide/tutorial/configureObject/configureObject.out[]
+----
 
 [[sec:configuring_arbitrary_objects_using_an_external_script]]
-=== Configuring arbitrary objects using an external script
+== Configuring arbitrary objects using an external script
 
 You can also configure arbitrary objects using an external script.
 
-++++
-<sample xmlns:xi="http://www.w3.org/2001/XInclude" id="configureObjectUsingScript" dir="userguide/tutorial/configureObjectUsingScript" title="Configuring arbitrary objects using a script">
-    <sourcefile file="build.gradle"/>
-    <sourcefile file="other.gradle"/>
-    <output args="-q configure"/>
-</sample>
-++++
+=== Example: Configuring arbitrary objects using a script
+
+[source.multi-language-sample,groovy]
+====
+.build.gradle
+----
+include::{samplesPath}/userguide/tutorial/configureObjectUsingScript/build.gradle[]
+----
+
+.other.gradle
```

Comment:
`other.gradle` file name is in the same line as `build.gradle`, which is wrong:

![image](https://user-images.githubusercontent.com/12689835/43069309-57a23680-8e9f-11e8-801e-be6a1ee8ed5f.png)

Reply:
Checked and fixed.